Lombiq.Analyzers.PowerShell/Rules/Measure-LineContinuation/Measure-LineContinuation.psm1

```diff
@@ -38,12 +38,12 @@ function Measure-LineContinuation
                     $PSItem.Kind -eq [System.Management.Automation.Language.TokenKind]::LineContinuation })
             {
                 $results += [Microsoft.Windows.Powershell.ScriptAnalyzer.Generic.DiagnosticRecord]@{
-                    "Extent" = $lineContinuationToken.Extent
-                    "Message" = 'Using backtick (line continuation) makes the code harder to read and' +
-                    ' maintain. Please consider using parameter splatting instead.'
-                    "RuleName" = "PSAvoidUsingLineContinuation"
-                    "RuleSuppressionID" = "PSAvoidUsingLineContinuation"
-                    "Severity" = "Warning"
+                    'Extent' = $lineContinuationToken.Extent
+                    'Message' = 'Using backtick (line continuation) makes the code harder to read and maintain.' +
+                    ' Please consider using parameter splatting instead.'
```

Comment:
Is this auto-formatted? Without any indentation in the second line of the concatenated string? 🙄

Reply:
Yes - surrounding it with parenthesis makes the auto-formatting give the expected result.